source/voting.rst

```diff
@@ -0,0 +1,96 @@
+.. _voting:
+
+############
+Voting Items
+############
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project developers’ mailing list, the issue tracker, and in the
+project meetings mentioned above. Occasionally, sensitive discussion occurs on
+a private list, but the project defaults to open meetings.
+
+The yt-project uses a “consensus seeking” process for making large,
+project-changing decisions. The
+group tries to find a resolution that has no open objections among project
+members. At any point during the discussion, any project member can call for
+a vote, which will conclude two weeks from the call for the vote. Any vote must
+be backed by a YTEP. If no option can gather two thirds of the votes cast, the
+decision is escalated to the Steering Committee, which in turn will use
+consensus seeking with the fallback option of a simple majority vote if no
+consensus can be found within a month. 
+This is what we hereafter may refer to as “the decision making process”.
+
+Decisions (in addition to adding project members and Steering Committee
+membership as above) are made according to the following rules (+1 is
+equivalently an approval vote on a pull request):
+
+Minor Documentation Changes
+---------------------------
+
+Minor Documentation changes, such as typo fixes, or addition / correction of a
+sentence: Requires +1 by a project member, no -1 by a project member (lazy
+consensus), happens on the issue or pull request page. Project members are
+expected to give “reasonable time” to others to give their opinion on the pull
+request if they’re not confident others would agree.
+
+Code Changes and Major Documentation Changes
+--------------------------------------------
```

Comment:
What context is this happening in? Has someone written an email to yt-dev proposing changes? Have they issued a PR?

Reply:
Good point! What standards do we want to have here? This is describing the process to merge a PR. Do we require an e-mail to yt-dev for all code changes? Or can it be either a discussion in an issue or an e-mail to yt-dev? Should we require that that discussion is linked in the open PR too?

Reply:
I don't think we should mandate an e-mail to yt-dev for all code changes, but we should mandate e-mails about major changes, including API breaks, big new features, or any other change of sufficient magnitude that a reviewer deems it necessary to ask the submitted to mail yt-dev and start a development discussion. Not all changes like this will require a YTEP but I think it's worth allowing for a formal process for asking to start a development discussion off github.

Reply:
Ok I've hopefully modified this. I also took out some of the requirements of having a YTEP.